crates/octez/src/async/protocol.rs

```diff
@@ -183,6 +199,25 @@ impl ProtocolParameterBuilder {
         json.insert("bootstrap_accounts".to_owned(), Value::from(&accounts));
         Ok(())
     }
+
+    fn merge_bootstrap_contracts(
+        &mut self,
+        json: &mut serde_json::Map<String, Value>,
+    ) -> anyhow::Result<()> {
+        let mut contracts = BootstrapContracts::default();
+        if let Some(value) = json.get("bootstrap_contracts") {
+            if let Some(existing_contracts) = value.as_array() {
+                for contract in existing_contracts {
+                    contracts.add_contract(BootstrapContract::try_from(contract)?);
+                }
+            }
+        }
+        for contract in self.bootstrap_contracts.contracts() {
+            contracts.add_contract(contract.clone());
+        }
```

Comment:
`impl BootstrapContracts` should define `fn merge(&mut self, bootstrap_contracts: BootstrapContracts)`

Reply:
Updated